apps/docs/plopfile.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = plop => {
+  plop.load('@not-govuk/app-plop-pack');
```

Comment:
With reference to my earlier comment: couldn't this file be replaced by a simple script in `package.json` like `"plop": "npx @not-govuk/app-plop-pack ..."` ?